resources/lang/en.json

```
@@ -8,5 +8,20 @@
     "Confirm Password": "Confirm Password",
     "Remember Me": "Remember Me",
     "Please confirm your password": "Please confirm your password",
-    "Forgot Your Password?": "Forgot Your Password?"
+    "Forgot Your Password?": "Forgot Your Password?",
+    "Name": "Name",
+    "Edit": "Edit",
+    "Delete": "Delete",
+    "Dashboard": "Dashboard",
+    "New": "New",
+    "Categories": "Categories",
+    "submit": "submit",
+    "Name is required!": "Name is required!",
+    "Successfull!": "Successfull!",
+    "Create successfull": "Create successfull",
+    "Update successfull": "Update successfull",
+    "Delete success": "Delete success",
+    "Not found category": "Not found category",
+    "name.unique": "Name is already existed",
+    "name.required": "Name is required"
```

```suggestion
    "successful": "Successful!",
    "create_successful": "Create successful",
    "update_successful": "Update successful",
    "delete_success": "Delete success",
    "category.not_found": "Not found category",
    "name.unique": "Name is already existed",
    "name.required": "Name is required"
```
trùng bản dịch cho required. sai chính tả, sửa lại key nhé.